seed.py

```diff
@@ -0,0 +1,44 @@
+from app import create_app, db
+from app.models.task import Task
+from app.models.goal import Goal
+from datetime import datetime
+
+from dotenv import load_dotenv
+load_dotenv()
+
```

Comment:
You don't need these lines here. When `python seed.py` is executed, your flask app will look to `app/__init__.py` and execute the code in create_app() on line 9 here.